### PR TITLE
add: fast data loading section

### PR DESCRIPTION
**Pull request by flower-of-the-bridges**
## Description

This PR introduces the concepts of Initial Load and Full Refresh in a dedicated page of the fast data section.
In this section are explained:
- when this operation are used
- best practice to perform them

## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

File: docs/fast_data/runtime_management/overview.mdx [hunk 1]
```diff
@@ -78,7 +78,7 @@ For more details on the Control Plane service configuration, please head over th
 ## Concepts
 
 Here are listed some concepts that are used on the Runtime Management solution. 
-This concepts are in addition the ones already provided by [Fast Data Section](/fast_data/glossary.mdx). 
+This concepts are in addition the ones already provided by [Fast Data Section](/fast_data/concepts/glossary.mdxx).
```
Comment (albertotessarotto) [inline docs/fast_data/runtime_management/overview.mdx]:
```suggestion
This concepts are in addition the ones already provided by [Fast Data Section](/fast_data/concepts/glossary.mdx). 
```

File: docs/fast_data/concepts/data_loading.mdx [hunk 2]
```diff
@@ -0,0 +1,472 @@
+---
+id: data_loading
+title: Fast Data Loading
+sidebar_label: Data Loading
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+__Data Loading__ is a fundamental process to introduce information from the Systems of Record:
```
Comment (albertotessarotto) [inline docs/fast_data/concepts/data_loading.mdx:10]:
```suggestion
__Data Loading__ is a fundamental process to introduce information from the Systems of Record to your Fast Data system:
```

File: docs/fast_data/concepts/data_loading.mdx [hunk 3]
```diff
@@ -0,0 +1,472 @@
...
+
+Then, if there exists a Single View Trigger Generator that consumes the messages produced into the `pr-update` topics of
+the projections, you have to check that message consuming is not up-and-running (either by using Runtime Management or setting replicas to zero).
+
+In this way, during the ingestion phase, strategies will not be executed. Once the ingestion procedure has been completed, you can restart
+the SVTG, so that strategies will generate the single view identifiers accordingly.
+
+##### Using Runtime Management
+
+With Runtime Management, is possible to manage Initial Load operations without altering your configuration.
```
Comment (albertotessarotto) [inline docs/fast_data/concepts/data_loading.mdx:63]: We may put a link to runtime management section

File: docs/fast_data/concepts/data_loading.mdx [hunk 4]
```diff
@@ -0,0 +1,472 @@
...
+
+From the Control Plane frontend, you have to check first that the Domain Pipeline that will consume messages
```
Comment (albertotessarotto) [inline docs/fast_data/concepts/data_loading.mdx:65]: Alternatively, here we may put a link to runtime management fast data section with control plane info

File: docs/fast_data/concepts/data_loading.mdx [hunk 5]
```diff
@@ -0,0 +1,472 @@
...
+is in a running state. Then, in the Single View pipeline that will receives data streams
```
Comment (albertotessarotto) [inline docs/fast_data/concepts/data_loading.mdx:66]:
```suggestion
is in a running state. Then, in the Single View pipeline that will receive data streams
```

File: docs/fast_data/concepts/data_loading.mdx [hunk 6]
```diff
@@ -0,0 +1,472 @@
...
+
+![Initial Load Runtime Management](img/data_loading_initial_load_runtime.png)
+
+Then, once the Initial Load has been completed and messages have been stored correctly on MongoDB, you can put the `Collected`
+stage back to the __Running State__ and let the underlying SVTG consuming the `pr-update` messages to generate the `trigger` messages.
+
+![Initial Load Runtime Management](img/data_loading_initial_load_runtime_restart_collected.png)
+
+:::tip
+If the Initial Load involves projections having large datasets, is suggested to __only__ restart the base projection, i.e. the projection
```
Comment (albertotessarotto) [inline docs/fast_data/concepts/data_loading.mdx:80]: may have a link to base projection documentation details?

File: docs/fast_data/concepts/data_loading.mdx [hunk 7]
```diff
@@ -0,0 +1,472 @@
...
+  infrastractural problems or misconfigurations.
+
+### Best Practices
+
+Before refreshing a single view collection, be sure that strategies have been disabled:
+this is to ensure that projections are not changing while the refresh operation is taking place,
+which benefits also the memory usage of your database infrastructure.
+
+Below you can find the instructions to setup your runtime based on the
+[Fast Data architecture](/fast_data/concepts/architecture.md#putting-it-all-together) you've deployed.
```
Comment (albertotessarotto) [inline docs/fast_data/concepts/data_loading.mdx:132]:
```suggestion
[Fast Data architecture](/fast_data/concepts/architecture.md#putting-it-all-together) you have deployed.
```

File: docs/fast_data/concepts/data_loading.mdx (hunk 6 above)
Comment (flower-of-the-bridges) [reply docs/fast_data/concepts/data_loading.mdx:80]: link added [here](https://github.com/mia-platform/documentation/pull/1391/commits/730e00d01bbf09241b3c687c05224456843836a7)

File: docs/fast_data/concepts/data_loading.mdx (hunk 4 above)
Comment (flower-of-the-bridges) [reply docs/fast_data/concepts/data_loading.mdx]: link added [here](https://github.com/mia-platform/documentation/pull/1391/commits/730e00d01bbf09241b3c687c05224456843836a7)

File: docs/fast_data/concepts/data_loading.mdx (hunk 3 above)
Comment (flower-of-the-bridges) [reply docs/fast_data/concepts/data_loading.mdx]: link added [here](https://github.com/mia-platform/documentation/pull/1391/commits/730e00d01bbf09241b3c687c05224456843836a7)

**Pull request merged**